std_msgs/Time msg_sent_time
float64 x_vel
float64 y_vel
std_msgs/Duration time_to_impact
